lib/base64.c

```diff
@@ -93,7 +93,7 @@ static size_t decodeQuantum(unsigned char *dest, const char *src)
 /*
  * Curl_base64_decode()
  *
- * Given a base64 NUL-terminated string at src, decode it and return a
+ * Given a base64 NULL-terminated string at src, decode it and return a
```

Comment:
I don't think is correct. We can call them "null-terminated" and get away with it, but NULL is a pointer. Strings are not NULL terminated. NUL is the character, the name of the first character in the ASCII character set.